libs/logger/inc/custom_sink.hpp

```diff
@@ -65,14 +64,8 @@ class tfc_systemd_sink : public base_sink<mutex> {
       payload = msg.payload;
     }
 
-    size_t length = payload.size();
-    // limit to max int
-    if (length > static_cast<size_t>(std::numeric_limits<int>::max())) {
-      length = static_cast<size_t>(std::numeric_limits<int>::max());
-    }
-
```

Comment:
I wonder what the reason was for this. I am making this note so that I can remember to check the max message size.